pyang/plugin.py

```diff
@@ -117,6 +129,9 @@ def setup_fmt(self, ctx):
         """
         return
 
+    # setup_xform is preferred to setup_fmt for transform plugins
+    setup_xform = setup_fmt
```

Comment:
I think setup_xform MUST be used by transform plugins.  I.e., define setup_xform as a separate method.  No reason to accept a transform plugin to be able to use setup_fmt.

Reply:
Agreed. I've committed and pushed a fix.